libs/api-interfaces/src/lib/aggregate-views/view-models/resources/media-items/contributor-and-role.interface.ts

```diff
@@ -0,0 +1,4 @@
+export interface IContributorAndRole {
+    contributorId: string;
```

Comment:
Remind Justin that he needs to deal with this on the front-end (until we have a Contributors model). For now
- have a contributors array in the config with this info
- use this info to 'join' in the contributor names